shared/common-adapters/web-view.native.js

```diff
@@ -15,9 +15,9 @@ const combineJavaScriptAndCSS = (injections?: WebViewInjections) =>
   document.body.appendChild(style)
   style.type = 'text/css'
   style.appendChild(document.createTextNode(\`${escape(injections.css)}\`))
-})()
+})();
 
-(function() {\` ${escape(injections.javaScript)} \`})()
+(function() { ${escape(injections.javaScript)} })();
```

Comment:
We weren't using injected JS on native side, so this extra quote didn't break anything. But might as well fix it ...